client-runtime/src/main/java/com/microsoft/rest/v2/http/HttpHeaders.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2.http;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * A collection of headers that will be applied to a HTTP request.
+ */
+public class HttpHeaders implements Iterable<HttpHeader> {
```

Comment:
I'd recommend doing a conversion within your implementation of HttpClient.sendRequestAsync(). That seems like a specific thing about RxNetty and not a generic concept that all HttpClient implementations would need. Thanks for the heads up about it though.